### DISCUSSION TITLE
Added: LDM Notes for Jan 9th, 2019

### DISCUSSION THREAD

**Discussion by agocke** [dotnet/csharplang#2147] [General]
https://github.com/dotnet/csharplang/blob/master/meetings/2019/LDM-2019-01-09.md

## Agenda 

1. GetAsyncEnumerator signature
2. Ambiguities in nullable array type syntax
2. Recursive Patterns Open Language Issues https://github.com/dotnet/csharplang/issues/2095

**Comment by Joe4evr** [comment]
> Warnings for non-nullable fields that are not explicitly initialized

Glad this was considered, albeit with the conclusion of not doing anything (for now). Guess the best option for the dev is proper documentation that explains when what fields are/aren't null, and to explicitly surpress the warning by way of:
```cs
public SomeClass Foo { get; private set; } = null!;
```

**Comment by agocke** [comment]
@Joe4evr Right, our guidance right now is to initialize with `= null!`;
